lib/local_authorities_importer.rb

```diff
@@ -0,0 +1,72 @@
+require 'json_downloader'
+
+class LocalAuthoritiesImporter
+  include JsonDownloader
+
+  DISTRICT = 'district'
+  COUNTY = 'county'
+  UNITARY = 'unitary'
+
+  LOCAL_AUTHORITY_MAPPING = {
+    "COI" => UNITARY,
+    "CTY" => COUNTY,
+    "DIS" => DISTRICT,
+    "LBO" => UNITARY,
+    "LGD" => UNITARY,
+    "MTD" => UNITARY,
+    "UTA" => UNITARY,
+  }
+
+  def self.import_from_mapit
+    new.authorities_from_mapit
+  end
+
+  def authorities_from_mapit
+    url = "#{MAPIT_BASE_URL}areas/#{local_authority_types}"
+    mapit_las = get_mapit_authorities(url)
```

Comment:
Is it worth updating Publisher to follow this guideline?  Possibly not if it's only for the short term... https://github.com/alphagov/publisher/blob/30b44b7417285589484d7e0d18c76329afd2920e/lib/local_interaction_importer.rb#L78